lib/chewy/type.rb

```diff
@@ -41,6 +41,18 @@ def self.type_name
       adapter.type_name
     end
 
+    # Returns the identifier value
+    #
+    def self.identify objects
+      case root_object
```

Comment:
root_object is always a Chewy::Fields::Root, how do you check it against other types?